FrontEndTest/FrontEndTest/ContentView.swift

```diff
@@ -10,7 +10,146 @@
 //
 
 import SwiftUI
+import Foundation
+import EventKit
+import CoreData
 
+
+func addWorkoutsToCalendar(workouts: [Workout], completion: ((_ success: Bool, _ error: NSError?) -> Void)? = nil) {
```

Comment:
I used to have it there but it opens too many connections to the calendar and bugs out.  If we had a LiftingProgram class it would make sense to post them all from that object.